packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -712,7 +712,7 @@ describe('<ButtonBase />', () => {
 
         expect(onClickSpy.callCount).to.equal(1);
         // defaultPrevented?
-        expect(onClickSpy.returnValues[0]).to.equal(true);
+        expect(onClickSpy.returnValues[0]).to.equal(false);
```

Comment:
I'm not following. This means the default isn't prevented which will cause scrolling when pressing Space. I thought the intention was to prevent scrolling?

Reply:
That was the keyUp, makes sense